src/components/ChatEntry.js

```diff
@@ -1,22 +1,28 @@
 import React from 'react';
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
 
 const ChatEntry = (props) => {
```

Comment:
Just a friendly reminder that we can also de-structure our `props` with `const ChatEntry = ( {myProp})` to have direct access to them!